Python/String/1120.py

```diff
@@ -0,0 +1,30 @@
+# Problema 1120 - URI - Strings - Nível 4
+
+# Inicializamos as variáveis, pois não existe Do While em python
```

Comment:
É possível fazer um "do while" com:

while True:
...
if d == '0' and n == '0':
break